runtime/integration-tests/src/utils/accounts.rs

```diff
@@ -224,7 +224,7 @@ pub fn default_accounts() -> Vec<Keyring> {
 
 /// Returns a Vector of default investor accounts
 pub fn default_investors() -> Vec<Keyring> {
-	(0..=50).map(Keyring::TrancheInvestor).collect()
+	(1..=2).map(Keyring::TrancheInvestor).collect()
```

Comment:
This reduces the time for each test case to more than a minute on my computer (around 70 seconds less per test case). I think we can enable again the `all` tag for these tests.